src/Orleans.Core/Statistics/ClientStatisticsManager.cs

```diff
@@ -3,47 +3,36 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.DependencyInjection;
-using Orleans.Runtime.Configuration;
 using Orleans.Serialization;
 using Orleans.Hosting;
 using Microsoft.Extensions.Options;
 using Orleans.Messaging;
-using Orleans.Statistics;
 
 namespace Orleans.Runtime
 {
     internal class ClientStatisticsManager : IDisposable
     {
         private readonly ClientStatisticsOptions statisticsOptions;
         private readonly IServiceProvider serviceProvider;
-        private readonly IHostEnvironmentStatistics hostEnvironmentStatistics;
-        private readonly IAppEnvironmentStatistics appEnvironmentStatistics;
         private readonly ClusterClientOptions clusterClientOptions;
-        private ClientTableStatistics tableStatistics;
         private LogStatistics logStatistics;
         private readonly ILogger logger;
-        private readonly ILoggerFactory loggerFactory;
-        private MonitoringStorageOptions storageOptions;
-        private string dnsHostName;
+        private readonly MonitoringStorageOptions storageOptions;
+        private readonly string dnsHostName;
         public ClientStatisticsManager(
             IOptions<MonitoringStorageOptions> storageOptions,
             SerializationManager serializationManager, 
             IServiceProvider serviceProvider,
-            IHostEnvironmentStatistics hostEnvironmentStatistics,
```

Comment:
why removing this two. Doesn't statistics still reports cpu, memory and all that ?

Reply:
Oh I guess these two are here only for constructing the default metric publisher.
Can you make sure statistics still reports `IHostEnvironmentStatistics ` and `IAppEnvironmentStatistics ` ? since those are pretty important statistics

Reply:
These values are only published to the metrics table, so we can remove them since they're no longer used.